package.json

```diff
@@ -23,8 +23,11 @@
     "pretest": "npm run lint",
     "test": "npm run jest",
     "jest": "cross-env MOCK_REDIS=1 jest",
-    "jest-watch":"cross-env MOCK_REDIS=1 jest --watch",
-    "start": "node src",
+    "jest-watch": "cross-env MOCK_REDIS=1 jest --watch",
+    "start": "node src/backend",
```

Comment:
None of these files should be changed in your branch as far as I can tell.  We've had people overwrite these already in the past few days, I'm not eager to repeat it.  If I'm wrong and you do need these to be updated, let me know why.